ext/intl/uchar/uchar.stub.php

```diff
@@ -0,0 +1,335 @@
+<?php
+
+class IntlChar
+{
+    /**
+     * @param int|string $codepoint
+     * @return bool|null
+     */
+    public static function hasBinaryProperty($codepoint, int $property) {}
+
+    /**
+     * @param int|string $codepoint
+     * @return array|null
+     */
+    public static function charAge($codepoint) {}
+
+    /**
+     * @param int|string $codepoint
+     * @return int|null
+     */
+    public static function charDigitValue($codepoint) {}
+
+    /**
+     * @param int|string $codepoint
+     * @return int|null
+     */
+    public static function charDirection($codepoint) {}
+
+    /** @return int|null */
+    public static function charFromName(string $characterName, int $nameChoice = IntlChar::UNICODE_CHAR_NAME) {}
+
+    /**
+     * @param int|string $codepoint
+     * @return int|string|null
+     */
+    public static function charMirror($codepoint) {}
+
+    /**
+     * @param int|string $codepoint
+     * @return string|null
+     */
+    public static function charName($codepoint, int $nameChoice = IntlChar::UNICODE_CHAR_NAME) {}
+
+    /**
+     * @param int|string $codepoint
+     * @return int|null
+     */
+    public static function charType($codepoint) {}
+
+    /**
+     * @param int|string $codepoint
+     * @return string|null
+     */
+    public static function chr($codepoint) {}
+
+    /**
+     * @param int|string $codepoint
+     * @return int|false|null
+     */
+    public static function digit($codepoint, int $radix = 10) {}
+
+    /**
+     * @param int|string $start
+     * @param int|string $limit
+     * @param callable $callback
```

Comment:
This one can also be moved to a proper type.

Reply:
And of course the arginfo should have been regenerated. I'm fixing these now.